worker/transcribee_worker/whisper_transcribe.py

```diff
@@ -123,39 +124,49 @@ def handle_new_segment(
     ctx = get_context(model_name)
 
     special_tokens = [
-        ctx.eot_token,
-        ctx.sot_token,
-        ctx.prev_token,
-        ctx.solm_token,
-        ctx.not_token,
-        ctx.beg_token,
+        ctx.eot_token,  # type: ignore
+        ctx.sot_token,  # type: ignore
+        ctx.prev_token,  # type: ignore
+        ctx.solm_token,  # type: ignore
+        ctx.not_token,  # type: ignore
+        ctx.beg_token,  # type: ignore
     ]
 
-    sampling = api.SamplingStrategies.from_strategy_type(api.SAMPLING_GREEDY)
+    sampling = api.SamplingStrategies.from_enum(api.SAMPLING_GREEDY)
     sampling.greedy.best_of = 5  # parameter stolen from whisper.cpp cli
-    params = api.Params.from_sampling_strategy(sampling)
-    params.no_context = (
-        False  # if False, feeds back already transcribed text back to the model
+    params = (
+        api.Params.from_sampling_strategy(sampling)
+        .with_no_context(
+            False
+        )  # if False, feeds back already transcribed text back to the model
+        .with_num_threads(4)
+        .with_language(lang_code)
+        .with_max_segment_length(120)  # parameter stolen from whisper.cpp cli
```

Comment:
Why is this changed from 60 to 120?

Reply:
Can we just set this to 0? I heard this will give best results?

Reply:
Where did you hear this?
Why does the whisper.cpp cli set it to `60` when one specifies `0` then? (ref https://github.com/ggerganov/whisper.cpp/blob/09e90680072d8ecdf02eaf21c393218385d2c616/examples/main/main.cpp#L590)

Reply:
Oops, got catch. I changed it during testing